### ISSUE TITLE
UX improvements regarding experience / level

### ISSUE THREAD

**Issue by domsson** [rhymu8354/OmniaRegna#291]
Currently, it is unclear how much experience one needs to reach the next level. A progress bar or simply an indication of required XP points for the next level might make sense.

**Comment by rhymu8354** [comment]
Yes, I did want to make some kind of tooltip and/or experience bar.
